transport_nantes/topicblog/views.py

```diff
@@ -504,6 +524,7 @@ def create_send_record(self,  slug: str, mailing_list: MailingList,
             recipient=recipient_user_object,
         )
         send_record.save()
+        logger.info(f"Created ")
```

Comment:
Bare "Created", out of context, is hard to figure what you created. Many things are, so you should probably mention that you created the send record. ( with its ID perhaps ?)

Reply:
Clearly something distracted me at just that moment...